index.js

```diff
@@ -1,55 +1,93 @@
 // PLUGIN_NAME: gulp-asciidoctor
-var through = require('through-gulp');
-var gutil = require('gulp-util');
-var asciidoctor = require('asciidoctor.js')();
+var through = require('through-gulp')
+var PluginError = require('plugin-error')
+var replaceExt = require('replace-ext')
+var asciidoctor = require('asciidoctor')()
```

Comment:
I think we should use `@asciidoctor/core`. The `asciidoctor` also includes the CLI which might not be needed here. It will reduce the dependencies tree.